### ISSUE TITLE
Augur of Bolas Bug

### ISSUE THREAD

**Issue by Coolhand2** [magefree/mage#4141]
From the log:

> 8:35 PM: Turn 9 computer (12 - 28)
> 8:35 PM: computer draws a card
> 8:35 PM: computer casts Augur of Bolas [546]
> 8:35 PM: computer puts Augur of Bolas [546] from stack onto the Battlefield
> 8:35 PM: Ability triggers: Augur of Bolas [546] - When Augur of Bolas [546] enters the battlefield, look at the top three cards of your library. You may reveal an instant or sorcery card from among them and put it into your hand. Put the rest on the bottom of your library in any order.
> 8:35 PM: computer casts Inside Out [8de] targeting Augur of Bolas [546] 
> 8:35 PM: computer draws a card
> 8:35 PM: computer puts Inside Out [8de] from stack into his or her graveyard
> 8:35 PM: computer attacks with 1 creature

No card was revealed on the Augur trigger, this stack trace occurred instead:
```
Game exception occurred: java.lang.IllegalStateException: Target wasn't handled. class:class mage.target.TargetCard
Server version: 1.4.26V7
mage.player.ai.ComputerPlayer.chooseTarget(ComputerPlayer.java:749)
mage.cards.a.AugurOfBolasEffect.apply(AugurOfBolas.java:117)
mage.abilities.AbilityImpl.resolve(AbilityImpl.java:187)
mage.abilities.TriggeredAbilityImpl.resolve(TriggeredAbilityImpl.java:101)
mage.game.stack.StackAbility.resolve(StackAbility.java:101)
mage.game.GameImpl.resolve(GameImpl.java:1347)
mage.game.GameImpl.playPriority(GameImpl.java:1298)
mage.game.turn.Step.priority(Step.java:87)
mage.game.turn.Phase.playStep(Phase.java:210)
mage.game.turn.Phase.play(Phase.java:115)
mage.game.turn.Turn.play(Turn.java:153)
mage.game.GameImpl.playTurn(GameImpl.java:816)
mage.game.GameImpl.play(GameImpl.java:725)
mage.game.GameImpl.start(GameImpl.java:689)
mage.server.game.GameWorker.call(GameWorker.java:60)
java.util.concurrent.FutureTask.run(FutureTask.java:266)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:748)
```

After the stack trace appeared (and the window dismissed), the AI decided to attack with their creature (included in the log, but doesn't seem overall relevant.)

**Comment by Coolhand2** [comment]
In case it becomes relevant, I was playing a Bogles deck, and the AI was playing an Inside Out Combo deck.

**Comment by Coolhand2** [comment]
The same issue occurred the next turn, same log, same cards. It appears the AI is casting Inside Out in response to the Augur trigger, then once the augur trigger occurs, the exception appears.

**Comment by JayDi85** [comment]
TargetCard and AI must be fixed by #6093
